test/controllers/api/v2/registration_tokens_controller_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class Api::V2::RegistrationTokensControllerTest < ActionController::TestCase
+  test 'user shall invalidate tokens for self' do
+    user = FactoryBot.create(:user)
+    FactoryBot.create(:jwt_secret, token: 'test_jwt_secret', user: user)
+    delete :destroy, params: { :id => user.id.to_s}
+    user.reload
+    assert_nil user.jwt_secret
+  end
+
+  test 'user with edit permission should be able to invalidate jwt for another user' do
+    setup_user 'edit', 'users'
+    user = users(:one)
+    FactoryBot.create(:jwt_secret, token: 'test_jwt_secret', user: user)
+    delete :invalidate_tokens, params: { :search => "id ^ (#{user.id})"}
+    user.reload
+    assert_nil user.jwt_secret
+  end
+
+  test 'user without edit permission should not be able to invalidate jwt for another user' do
+    User.current = users(:one)
+    user = users(:two)
+    FactoryBot.create(:jwt_secret, token: 'test_jwt_secret', user: user)
+    delete :invalidate_tokens, params: { :search => "id ^ (#{user.id})"}
+    user.reload
+    assert_response :forbidden
+  end
+
+  test 'invalidating jwt should fail without search params' do
+    setup_user 'edit', 'users'
+    user = users(:two)
+    FactoryBot.create(:jwt_secret, token: 'test_jwt_secret', user: user)
+    delete :invalidate_tokens
+    user.reload
+    assert_response :error
+  end
```

Comment:
We expect an exception to be raised in this case, correct? If so, the test should be updated to reflect that.

Reply:
I think using `assert_response :error` is the way to handle that, do you have any specific assertion on mind

Reply:
You can check if an exception was raised.

Reply:
@girijaasoni How did you resolve this?